crm-project/src/main/java/CRM/entity/requests/AttributeRequest.java

```diff
@@ -0,0 +1,15 @@
+package CRM.entity.requests;
+
+import CRM.entity.Board;
+import lombok.*;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Getter
+@Setter
+@ToString
+public class AttributeRequest {
+    private Board board;
```

Comment:
Request entities only accept primitive data e.g. Integer, String, Long, Boolean etc...
This attribute has to get:
- private Long boardId;

Reply:
fixed